### ISSUE TITLE
Correction Suggestion for Concurrent Merkle Tree Component in app/components/account/ConcurrentMerkleTreeCard.tsx

### ISSUE THREAD

**Issue by philetos** [solana-labs/explorer#260]
Hello,

I would like to bring a potential correction to your attention regarding the Concurrent Merkle Tree component located in the file `app/components/account/ConcurrentMerkleTreeCard.tsx` in the project. Please correct me if I am wrong, but I believe there are unnecessary subtraction and addition operations in two specific table rows that display the current number of leaves and the remaining leaves.

Currently, the code snippet in question appears as follows:

```typescript
<tr>
    <td>Current Number of Leaves</td>
    <td className="text-lg-end">
        <span className="text-monospace">{rightMostIndex - 1}</span>
    </td>
</tr>
<tr>
    <td>Remaining Leaves</td>
    <td className="text-lg-end">
        <span className="text-monospace">{Math.pow(2, treeHeight) - rightMostIndex + 1}</span>
    </td>
</tr>
```

If my understanding is correct, I would like to propose a modification to improve the accuracy of the displayed values by removing these unnecessary operations.

In the first table row, subtracting 1 from rightMostIndex seems redundant since rightMostIndex already represents the index of the rightmost leaf. Therefore, the subtraction can be omitted:

```typescript
<tr>
    <td>Current Number of Leaves</td>
    <td className="text-lg-end">
        <span className="text-monospace">{rightMostIndex}</span>
    </td>
</tr>
```

Similarly, in the second table row, adding 1 to rightMostIndex might not be necessary. I suggest the following modification:

```typescript
<tr>
    <td>Remaining Leaves</td>
    <td className="text-lg-end">
        <span className="text-monospace">{Math.pow(2, treeHeight) - rightMostIndex}</span>
    </td>
</tr>
```

By making these adjustments, we can present the correct values in a more straightforward manner, eliminating any potential confusion.

Please review this suggestion and let me know if I have misunderstood any aspects of the code. Your feedback and insights are highly appreciated.

Thank you for your attention to this matter!

Kind regards

<img width="1153" alt="Screen Shot 2023-06-24 at 5 14 56 PM" src="https://github.com/solana-labs/explorer/assets/794651/6b969267-51d3-439d-9e86-f8840c4e5c8c">

**Comment by theCyberFunk** [comment]
Hi, opened the #261 pr to address this issue.

**Comment by ngundotra** [comment]
Thanks for flagging! Yes I made an error when I committed this earlier. Thanks for putting out PR @theCyberFunk

**Comment by ngundotra** [comment]
Closed by #261
